kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiConnection.java

```diff
@@ -1366,4 +1380,8 @@ public String getEngineName() {
   public String getEngineUrl() {
     return engineUrl;
   }
+
+  public String getResultCodec() {
```

Comment:
the result codec should be operation level instead of connection, if you want this to a temporary solution, please avoid using public.

Reply:
the same with https://github.com/apache/incubator-kyuubi/pull/3803#discussion_r1033112667

Reply:
it's fine, but can we use package instead of public here?